src/transformation.js

```diff
@@ -560,6 +561,11 @@ class Transformation extends TransformationBase {
   /*
     Transformation Parameters
   */
+  accessibility(value) {
+    this.chain();
+    return this.effect(ACCESSIBILITY_MODES[value]);
```

Comment:
It's currently only effects, what if tomorrow we will have something else?

Reply:
this is now handled by cloudinary.url() and will be updated as needed